sympy/external/importtools.py

```diff
@@ -28,6 +28,23 @@ def __sympy_debug():
     WARN_NOT_INSTALLED = True
 
 
+_component_re = re.compile(r'(\d+ | [a-z]+ | \.)', re.VERBOSE)
+
+def version_tuple(vstring):
+    # Parse a version string to a tuple e.g. '1.2' -> (1, 2)
+    # Simplified from distutils.version.LooseVersion which was deprecated in
+    # Python 3.10.
+    components = []
+    for x in _component_re.split(vstring):
+        if x and x != '.':
+            try:
+                x = int(x)
+            except ValueError:
+                pass
+            components.append(x)
+    return components
```

Comment:
`return tuple(components)`?

Reply:
Yeah, good point. It doesn't matter if it's only used in version comparison but I suppose in principle it could be used in something like a set. Mainly it just means the name is misleading.